.github/workflows/ci.yml

```diff
@@ -0,0 +1,137 @@
+name: CI
+
+concurrency:
+  group: ci-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
```

Comment:
We don't have these branches in this repo

Reply:
This is consistent to other repositories, you can change later alognside others.

Reply:
I would rather choose to not introduce it if it is not needed/used.

Reply:
If possible, I'd lean to stripping the CI down to bare minimum of what we know is hooked up to something because it will run as part of the marge.  But we'll also plan to make a more involved cleanup pass later, aka https://github.com/infinyon/nats-connector/issues/3.

Reply:
stripped